Gopkg.lock

```diff
@@ -289,6 +289,12 @@
   revision = "150dc57a1b433e64154302bdc40b6bb8aefa313a"
   version = "v1.0.0"
 
+[[projects]]
+  branch = "v1"
+  name = "gopkg.in/check.v1"
```

Comment:
Our errors and validate unit tests are already using github.com/stretchr/testify as a test toolkit, can we converge on that or is there something in check.v1 that you like? check.v1 seems un-maintained (only a handful of checkins in 2016, none in 2017).

Reply:
Cool. Remove gocheck in favor of testify.